astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -20,8 +20,10 @@
 from astropy.visualization.wcsaxes.utils import get_coord_meta
 from astropy.visualization.wcsaxes.transforms import CurvedTransform
 
-MATPLOTLIB_LT_21 = Version(matplotlib.__version__) < Version("2.1")
-MATPLOTLIB_LT_22 = Version(matplotlib.__version__) < Version("2.2")
+mpl_version = Version(matplotlib.__version__)
+MATPLOTLIB_LT_21 = mpl_version < Version("2.1")
+MATPLOTLIB_LT_22 = mpl_version < Version("2.2")
+NOT_MATPLOTLIB_33 = mpl_version.major != 3 or mpl_version.minor != 3
```

Comment:
Can you make this ``MATPLOTLIB_33`` and write ``not MATPLOTLIB_33`` in the skipif below?